FirstProject/assets/evaluation.less

```diff
@@ -0,0 +1,54 @@
+//Evaluation less file
+
+.row {
+
+  .navigation_bar {
+    border-bottom: 2px solid darkgrey;
+    padding-top: 12px;
+    padding-bottom: 14px;
+  }
+
+  .logo_image {
+    width: 40%;
+  }
+}
+
+.col-md-4 {
+
+  .footer_description {
+    padding: 25px 0 12px 0;
+    font-weight:bold;
+    font-size:16px;
+    color: gray;
+  }
+}
+
+.row{
+
+  .container_table {
+    margin-top: 120px;
+  }
+}
+
+.navigation_bar {
+
+  .col-md-4 {
+    text-align:center;
+  }
+
+  .evaluation_button {
+    padding-top: 4px;
+
+    .new_evaluation {
```

Comment:
Could you describe to me what the targeted markup for this selector (including relevant ancestors) looks like?